pyproject.toml

```diff
@@ -95,3 +81,45 @@ torchseg = ["py.typed"]
 
 [tool.setuptools.packages.find]
 include = ["torchseg*"]
+
+[tool.ruff]
+exclude = [
+    # Extra files
+    "data/",
+    "images/",
+    "logo/",
+    "logs/",
+    "output/",
+
+    # Docs
+    "docs/src/",
+
+    # Spack
+    ".spack-env/",
+
+    # Python
+    "build/",
+    "dist/",
+    ".cache/",
+    ".mypy_cache/",
+    ".pytest_cache/",
+    "__pycache__/",
+    "*.egg-info/",
+
+    # Git
+    ".git/",
+    ".github/",
+]
+
+line-length = 88
+indent-width = 4
+target-version = "py311"
+
+[tool.ruff.format]
+skip-magic-trailing-comma = true
+
+[tool.ruff.lint]
+ignore = ["E203"]
```

Comment:
I think you need something like:
```toml
[tool.ruff.lint]
extend-select = [
  "UP",  # pyupgrade
]
```
to add additional style tools besides the default (flake8).

Reply:
You're right. I just went through the settings and I'm going to update `.pyproject.toml` with the new settings. It seems like most of what we want is already on by default which is nice for reducing the amount of lines in `.pyproject.toml`